csharp/HOLMS.Platform/HOLMS.Platform.Tests/Tests/EntityDTOTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using HOLMS.Types.Money.Accounting;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HOLMS.Platform.Tests.Tests {
+    class EntityDTOTests {
+        [Test]
+        public void EqualsOperatorComparesEntityNotIndicatorToNull() {
+            var line = new AccountingTransactionLine() { };
+            var actual = line == null;
```

Comment:
This reads a little rough. Maybe put parens around it like this: `var actual = (line == null)`

Another case where no, it's not strictly required, but it emphasizes that you're expecting to assign a boolean.